config/chain_config.js

```diff
@@ -0,0 +1,55 @@
+class ChainConfig {
```

Comment:
Another proposal, we can have a configuration file with default options saved in repo and use it to fill config object.